lib/main.coffee

```diff
@@ -113,13 +117,18 @@ Notifications =
       unless timeSpan < @duplicateTimeDelay and notification.isEqual(@lastNotification)
         @notificationsElement.appendChild(atom.views.getView(notification).element)
         @notificationsLog?.addNotification(notification)
+        @statusBarManager?.addNotification(notification)
     else
       @notificationsElement.appendChild(atom.views.getView(notification).element)
       @notificationsLog?.addNotification(notification)
+      @statusBarManager?.addNotification(notification)
 
     notification.setDisplayed(true)
     @lastNotification = notification
 
+  consumeStatusBar: (statusBar) ->
+    @statusBarManager = new NotificationsStatusView(statusBar, @duplicateTimeDelay)
```

Comment:
`notificationsStatusView`

Reply:
Shouldn't that be uppercase since it is a class like `NotificationElement` and `NotificationsLog`

Reply:
oh sorry, I meant changing `statusBarManager` to `notificationsStatusView`.

Reply:
that makes more sense